src/features/access/AccessConstants.ts

```diff
@@ -0,0 +1,3 @@
+const permissionStatusURL = 'http://arkhn.com/fhir/cohort360/StructureDefinition/permission-status'
```

Comment:
CONSTANTS_NAME_ARE_LIKE_THIS
put in constants.ts
rename PERMISSION_STATUS_STRUCTURE_DEF_URL